content/en/service_management/service_level_objectives/guide/slo_types_comparison.md

```diff
@@ -0,0 +1,45 @@
+
+## Comparison chart
+
+|                                                                       | **Metric-based SLO**                                                                                                                      | **Monitor-based SLO**                                                                                                                                               | **Time Slice SLO**                                                                   |
+|-----------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------|
+| **Supported data types**                                              | Metrics with type of count, rate, or distribution                                                                                         | Metric Monitor types, Synthetic Monitors, and Service Checks                                                                                                        | All metric types (including gauge metrics)                                           |
+| **Group functionality**                                               | Unlimited groups per SLO                                                                                                                  | Limited to 20 monitor groups per SLO                                                                                                                                | Up to 5,000 groups per SLO                                                           |
+| **SLO details side panel History (up to 90 days of historical data)** | Can set custom time windows to view SLO info                                                                                              | Cannot set custom time windows to view SLO info                                                                                                                     | Can set custom time windows to view SLO info                                         |
+| **SLO alerting ([Error Budget][1] or [Burn Rate][2] Alerts)**         | Available for all metric-based SLOs                                                                                                       | Available for SLOs based on Metric Monitor types only (not available for Synthetic Monitors or Service Checks)                                                      | Not available for Time Slice SLOs                                                    |
```

Comment:
Can we keep "Not yet available for Time Slice SLOs" for the alerting item? I included "yet" since it's something we will add in soon

Reply:
We try not to use [temporal words](https://github.com/DataDog/documentation/pull/20888) or make promises for future features. Could we  change it when it becomes available?

Reply:
Sure but then let's remove it from the other item (Data Sources). 

My other comment was misplaced so adding it here: For the 2 items that say "Not yet available for Time Slice SLOs", can we add an asterisks/number with a note that says "Time Slice SLOs are currently in public beta. Support for Alerting and the SLO Data Source will be added for Time Slice SLOs in the future." ? I know we don't want to make promises but I also want to highlight this is in public beta and these limitations will be addressed - lmk the best way to frame this

Reply:
Between the two, the "yet" language is the better option. I'll add it back in.